packages/terra-overlay/src/LoadingOverlay.jsx

```diff
@@ -59,7 +59,7 @@ const LoadingOverlay = ({
   delete customProps.onRequestClose;
 
   return (
-    <Overlay {...customProps} className={cx('loading-overlay', customProps.className)} rootSelector={rootSelector}>
+    <Overlay {...customProps} className={cx('loading-overlay', customProps.className)} rootSelector={rootSelector} aria-live="polite">
```

Comment:
should we toggle `aria-busy` based on `isOpen`?

Reply:
Good point. If we need to do that, then shouldn't it be set on the parent node or the container that is _literally_ loading / busy?

Reply:
Yeah, we probably need documentation that points this out for containers

Reply:
Right. 

@tbiethman @mmalaker What are your thoughts on this?

Reply:
Would we gain anything by marking it with `aria-busy`? Based on what I'm reading [here](https://www.w3.org/TR/wai-aria-1.0/states_and_properties#aria-busy), it seems as though we might be getting the same functional result by inerting the content behind the overlay (like we do already).

Reply:
I tried adding it to the container and it didn't make a difference to me.

Reply:
@scottwilmarth do you know if aria-busy is something we could/should generically set on content underneath our overlays? Even if that content is already made inert and unaccessible?